src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAlone.java

```diff
@@ -27,6 +27,13 @@ void method2(java.util.HashSet<String> set) {
             put("polygene", "lubricants");
             put("alpha", "betical");
         }}; //NO violation
+
+        java.util.Map<String, String> map3 = new java.util.LinkedHashMap<String, String>() {{
```

Comment:
Please put this test case near the bottom of the file so line numbers in test aren't changed.

Reply:
changed as request